app/templates/login.html

```diff
@@ -14,31 +14,52 @@
     <nav class="navbar navbar-light bg-light justify-content-between"">
       <a class="navbar-brand" href="#">
         <img src="{{ url_for('static', filename='img/cachengo_logo.png') }}" height="30" width="119" alt="">
-        <img src="{{ url_for('static', filename='img/2000px-Orange_logo.svg.png') }}" height="60" width="60" alt="">
       </a>
       <div class="navbar-nav">
         <ul class="nav navbar-nav navbar-right">
           <button class="btn btn-dark btn-lg" type="button" onclick="relocate('/sign_up')">Sign Up</button>
         </ul>
       </div>
     </nav>
-    <div class="app container">
-        <div class="row">
-          <div class="col-lg-4" align="center">
-            <a href="#" id="start-camera">Touch here to start the camera</a>
-            <video id="camera-stream" height=400 width=400 class="invisible"></video>
-            <p id="error-message"></p>
-            <button id="take-photo" class="btn btn-dark btn-lg" type="button" onclick="login()">Login</button>
-
-            <!-- Hidden canvas element. Used for taking snapshot of video. -->
-            <canvas></canvas>
+    
+    <div id="app container">
+
+      <div class=" mt-5 mx-3 row justify-content-center">
+        <div class='col-12'>
+          <h3 class="text-center">Sign In</h3>
+        </div>
+
+        <div class="col-lg-6 ">
+          <div id="vid_container">
+            <video id="video" class="" autoplay playsinline></video>
+            <canvas id="hiddenCanvas" class='hidden'></canvas>
           </div>
         </div>
+
+        <div class='col-12'></div>
+
+        <div class=''>
+          <button id="take-photo" class="btn btn-light btn-lg" type="button" onclick="login()">Login</button>
+        </div>
+      </div>
+
     </div>
 
+
     <script src="{{ url_for('static', filename='js/ext/jquery.min.js') }}"></script>
     <script src="{{ url_for('static', filename='js/ext/bootstrap.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='js/int/camera.js') }}"></script>
+    
+    <script src="{{ url_for('static', filename='js/int/DetectRTC.min.js') }}"></script>
+    <script src="{{ url_for('static', filename='js/int/adapter.min.js') }}"></script>
+    <script src="{{ url_for('static', filename='js/int/screenfull.min.js') }}"></script>
+    <script src="{{ url_for('static', filename='js/int/howler.core.min.js') }}"></script>
```

Comment:
Are all of these modules actually required? DetectRTC is obviously used but screenfull doesn't seem useful. Also, they shouldn't be place under "js/int/" they should be under "js/ext".